proposals/XXXX-Adding-new-parameters-for-transport-switching.md

```diff
@@ -0,0 +1,122 @@
+# Adding new parameters for transport switching
+
+* Proposal: [SDL-XXXX](XXXX-Adding-new-parameters-for-transport-switching.md)
+* Author: [Kazuki Sugimoto(Nexty)](https://github.com/Kazuki-Sugimoto)
+* Status: **Awaiting Review**
+* Impacted Platforms: [ Core / iOS / Java Suite / RPC / HMI ]
+
+## Introduction
+
+This proposal is to add USB information and BT information to `deviceInfo` that is notified by `RegisterAppInterface` and `OnAppRegistered`, the purpose is to enable transport switching during a multiple transport connection if it is determined that they are the same device.
+
+## Motivation
+
+Currently, if an Android device makes a USB connection during BT connection, the transport cannot be switched because the HU cannot specify the device. For the HU to switch transport, it is necessary to determine whether or not they are the same device. Therefore, USB information and BT information are added to the device information to identify the device.
```

Comment:
```suggestion
Currently, if an Android device makes a USB connection during BT connection, the transport cannot be switched because the Head Unit (HU) cannot specify the device. For the HU to switch transports, it is necessary to determine whether or not they are the same device. Therefore, USB information and BT information are added to the device information to identify the device.
```